### PR TITLE
Import exchange integration doc and add redirects (DOCS-110, too)

### PR DESCRIPTION
**Pull request by ronny-mysten**
## Description 

Importing the exchange integration doc and updating links.

## Test Plan 

Local

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: docs/content/guides/operator/exchange-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+The response can include a large number of events. Add pagination to the response using the `nextCursor` key in the request. You can determine the corresponding `txDigest` and `eventSeq` from the `id` field of a transaction.
+
+You can add the `txDigest` value instead of the first `null` within the `params`. The second `null` is an integer that defines how many results (up to 1000) to return and the `true` means ascending order. You can use the `nextCursor` so the response starts from a desired point.
+
+The `id` field of any transaction looks like:
+```bash
+"id": {
+         "txDigest": "GZQN9pE3Zr9ZfLzBK1BfVCXtbjx5xKMxPSEKaHDvL3E2",
+         "eventSeq": 6019
+       }
```
Comment (amnn) [inline docs/content/guides/operator/exchange-integration.mdx:114]: nit: whitespace.

File: docs/content/guides/operator/exchange-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+## SUI balance transfer
+
+To transfer a specific amount of SUI between addresses, you need a SUI token object with that specific value. In Sui, everything is an object, including SUI tokens. The amount of SUI in each SUI token object varies. For example, an address could own 3 SUI tokens with different values: one of 0.1 SUI, a second of 1.0 SUI, and a third with 0.005 SUI. The total balance for the address equals the sum of the values of the individual SUI token objects, in this case, 1.105 SUI.
+
+You can merge and split SUI token objects to create token objects with specific values. To create a SUI token worth .6 SUI, split the token worth 1 SUI into two token objects worth .6 SUI and .4 SUI.
+
+To transfer a specific amount of SUI, you need a SUI token worth that specific amount. To get a SUI token with that specific value, you might need to split or merge existing SUI tokens. Sui supports several methods to accomplish this, including some that do not require you to manually split or merge coins.
+
+## Sui API operations for transfers
```
Comment (amnn) [inline docs/content/guides/operator/exchange-integration.mdx:144]: This section is quite out of date, as these operations are not long for  this world -- can we add it as a task to revisit this with more modern, PTB or CLI based instructions here?

**Review by amnn — APPROVED** [review]

File: docs/content/guides/operator/exchange-integration.mdx (hunk 1 above)
Comment (ronny-mysten) [reply docs/content/guides/operator/exchange-integration.mdx:114]: Of the spurious variety, even.

File: docs/content/guides/operator/exchange-integration.mdx (hunk 2 above)
Comment (ronny-mysten) [reply docs/content/guides/operator/exchange-integration.mdx:144]: DOCS-112

**Pull request merged**